src/experiments/77.instanced-grass/shaders.ts

```diff
@@ -0,0 +1,163 @@
+export const grassVertexShader = `
```

Comment:
Add glsl as a comment to trigger the linter, you can install the glsl-literal package to see it in action:

https://marketplace.visualstudio.com/items?itemName=boyswan.glsl-literal

```suggestion
export const grassVertexShader = /* glsl */`
```